### DISCUSSION TITLE
Version 1.4

### DISCUSSION THREAD

**Discussion by Alygnt** [Alygnt/Clifty#16] [Announcements]
## NPhisher is now Clifty

# Features:
- Added new fonts and colors
- Keep track of your options that you chose
- Network check before starting tunneler
- Changed type from password from the number on a few otp pages
- Display users IP in the main menu
- Custom redirect Url
- Improved assets management
- Added download all sites
- Network check before downloading the site
- Colours and UI are just awesome now

# Bugs fixed :
- Instagram OTP phishing page
- Network check error
- #13 fixed
- 

# Sites :
## Added
- None
## Updated
- Instagram
- Jio router

### **[View Change log ](https://github.com/Alygnt/Clifty/compare/v_1.3...v_1.4)**

<hr /><em>This discussion was created from the release <a href='https://github.com/Alygnt/Clifty/releases/tag/v_1.4'>Version 1.4</a>.</em>

**Comment by aryansingh100aryan** [comment]
Hello sir, ngrok and Cloudflared not working...
Plz Provide New Update with fixes ..
![236729841-30c4b6c4-77dd-439b-bf6a-d9098dd45390.jpg](https://github.com/Alygnt/Clifty/assets/145359738/5cffde55-34d7-4c4f-b5db-c84a424ee32e)

**Reply by Alygnt** [reply]
Fine, Started working on it
